### PR TITLE
Chek html

### PR DESCRIPTION
**Pull request by YurySuvalau**

### CODE REVIEW

File: src/test/java/htmlFindElements.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.Select;
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+public class htmlFindElements {
```
Comment (vchesnakoutest) [inline src/test/java/htmlFindElements.java]: Name for java classes should start from Capital letter.

File: src/test/java/htmlFindElements.java [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    WebDriver driver;
+    private static final String Mysite = "C:\\Users\\Бубята\\IdeaProjects\\HerokuApp\\src\\test\\resources\\Test.html";
```
Comment (vchesnakoutest) [inline src/test/java/htmlFindElements.java]: Name for constants should be in UPPER case: MYSITE or MY_SITE

File: src/test/java/htmlFindElements.java [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+    public void initMethod() {
+        System.setProperty("webdriver.chrome.driver", "src/test/resources/chromedriver.exe");
+        driver = new ChromeDriver();
+        driver.manage().window().maximize();
+    }
+    @Test
+    public void htmlFindElementTable() {
+        driver.get(Mysite);
+        String actualResult = driver.findElement(By.id("table01")).getAttribute("id");
+        Assert.assertEquals(actualResult, "table01");
```
Comment (vchesnakoutest) [inline src/test/java/htmlFindElements.java]: Test cases should NOT verify html attributes. You can verify if table is displayed on the page for example:
boolean actualResult = driver.findElement(By.id("table01")).isDisplayed();

File: src/test/java/htmlFindElements.java [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+        driver.quit();
+    }
+
+    @Test
+    public void htmlFindElementInput() {
+        driver.get(Mysite);
+        String actualResult = driver.findElement(By.name("email")).getAttribute("name");
+        Assert.assertEquals(actualResult, "email");
```
Comment (vchesnakoutest) [inline src/test/java/htmlFindElements.java]: here you're also checking html attribute, please fix it

File: src/test/java/htmlFindElements.java [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+        String actualResult = driver.findElement(By.id("ring")).getText();
+        Assert.assertEquals(actualResult, "The ring!");
+        driver.quit();
+    }
+
+    @Test
+    public void htmlFindElementImg() {
+        driver.get(Mysite);
+        String actualResult = driver.findElement(By.xpath("/html/body/img")).getAttribute("alt");
+        Assert.assertEquals(actualResult, "The house");
```
Comment (vchesnakoutest) [inline src/test/java/htmlFindElements.java]: here you're also checking html attribute, please fix it

File: src/test/resources/Test.html [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+     alt="The house" width="500" height="400">
+<h4 id="h4-7">
+    №7 Create a link
+
+</h4>
+<a href="https://google.com">Visit Google for search!</a>
+</body>
+
+
+</html>
```
Comment (vchesnakoutest) [inline src/test/resources/Test.html:123]: Good page! 👍

File: src/test/java/htmlFindElements.java [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+    }
+    @Test
+    public void htmlFindElementLink() {
+        driver.get(Mysite);
+        String actualResult = driver.findElement(By.xpath("/html/body/a")).getText();
+        Assert.assertEquals(actualResult, "Visit Google for search!");
+        driver.quit();
+    }
+
+}
```
Comment (vchesnakoutest) [inline src/test/java/htmlFindElements.java]: All other test cases are good 👍

File: src/test/java/htmlFindElements.java (hunk 1 above)
Comment (YurySuvalau) [reply src/test/java/htmlFindElements.java]: done

File: src/test/java/htmlFindElements.java (hunk 2 above)
Comment (YurySuvalau) [reply src/test/java/htmlFindElements.java]: done

File: src/test/java/htmlFindElements.java (hunk 4 above)
Comment (YurySuvalau) [reply src/test/java/htmlFindElements.java]: done

File: src/test/java/htmlFindElements.java (hunk 3 above)
Comment (YurySuvalau) [reply src/test/java/htmlFindElements.java]: fix

File: src/test/java/htmlFindElements.java (hunk 5 above)
Comment (YurySuvalau) [reply src/test/java/htmlFindElements.java]: fix

**Review by vchesnakoutest — APPROVED** [review]

**Pull request merged**